include/aspect/material_model/latent_heat_melt.h

```diff
@@ -230,6 +198,14 @@ namespace aspect
                                   const double pressure,
                                   const std::vector<double> &compositional_fields,
                                   const Point<dim> &position) const;
+
+        virtual
+        double
+        entropy_derivative ( const double temperature,
```

Comment:
This function was very long, I am using it as a private member just for this class.